misc/python/materialize/cli/mzcompose.py

```diff
@@ -18,9 +18,10 @@
 from pathlib import Path
 from typing import IO, List, Optional, Sequence, Text, Tuple
 
-from materialize import errors, mzbuild, mzcompose, spawn, ui
```

Comment:
This file is getting changed by `bin/pyfmt` for some reason, even though I didn't change it